sdk/cosmosdb/Azure.ResourceManager.CosmosDB/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.4.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.4.0-beta.2 (2023-05-22)
```

Comment:
Can release with the coming refresh for model factory

Reply:
reset the release date